zcash_test_vectors/transaction_v6.py

```diff
@@ -87,8 +87,10 @@ def __init__(self, rand, anchor_orchard, proofs_orchard, is_coinbase):
         # There must always be a non-zero number of Action Descriptions in an Action Group.
         for _ in range(rand.u8() % 4 + 1):
             self.vActionsOrchard.append(OrchardZSAActionDescription(rand))
-        # Three flag bits are defined, we set enableZSA to true.
-        self.flagsOrchard = (rand.u8() & 7) | 4
+        # Three flag bits are defined, ensure the other bits are zero by masking.
```

Comment:
maybe `ensure we use only the 3 first bits (mask 0b00000111)`

Reply:
Done.